### DISCUSSION TITLE
Training a subject and a style on the same model

### DISCUSSION THREAD

**Discussion by aniOfri** [TheLastBen/fast-stable-diffusion#1172] [Q&A] [answered]
I've been trying for about a week now to train a model which has my face and a style,
I tried training the instances together, separately (the subject first and then the style), starting with the subject for a couple hundred steps and then adding the style images and even just checkpoint merging to no avail.

When training two models one for each (subject/style) the results are perfect, together... no so much.

Any suggestions?

**Comment by TheLastBen** [comment, marked as answer]
training is subjective, some styles/faces require a lot of steps and some don't, so sometimes when you mix two instances with different requirements, you can end up overfitting one over the other, so it's more about trial and error for each instance, I recommend to start with a small amount of steps, then slowly increase, once you reach a good result for one subject, remove its instance images, and leave the others, then resume training.
